packages/library-detection/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "types": ["chrome"]
+  },
+  "include": [
+    "src",
+    "../../node_modules/@types/jest/index.d.ts",
+    "src/**/*.json"
+  ],
```

Comment:
```suggestion
  ],
  "exclude": ["**/tests/**/*.ts", "dist/**", "dist-types/**"]
```

Reply:
Exclude `"**/tests/**/*.ts"` ?

Reply:
Yes, otherwise when this module is used in the CLI where TSC is used to build JS files. JS versions of the test file will also be generated.